oaknational/Oak-Web-Application
# CTC-48: add phonic outcomes to lesson overview page header

## Description

Music year: 1997

- Renders phonic outcomes on the lesson overview header, updating relevant schema 
- Replaces deprecated flex and box components with comp lib versions

## Issue(s)

Fixes #CTC-48

### Acceptance Criteria:

- [ ]  Phonics outcome to be displayed on lesson overview page below the normal lesson outcome
- [ ]  Character limit of 190

## How to test

1. Go to https://deploy-preview-2965--oak-web-application.netlify.thenational.academy/teachers/beta/lessons/des-auteurs-francophones-perfect-tense-with-etre
2. You should see phonic outcomes on the mfl headers 


## Screenshots

How it should now look:

<img width="1297" alt="Screenshot 2024-11-06 at 16 46 35" src="https://github.com/user-attachments/assets/91e797f9-a74f-4a28-8156-57e1c8354547">



## Checklist
